wire-ios/WireUITests/WireAuthenticationTests.swift

```diff
@@ -53,7 +37,6 @@ final class WireAuthenticationTests: XCTestCase {
 
     @MainActor // note: comment @MainActor to use recorder
     func test_Login_withEmail() throws {
-        throw XCTSkip("This should be fixed once SSO is merged")
```

Comment:
question: hmm, so does it uses LoginCredentials.email? This was reading some sample account before from environment. Maybe we don't need that if we always create accounts on the fly?